test/audits/manifest/icons-192.js

```diff
@@ -40,7 +40,42 @@ describe('manifest: icons-192 audit', () => {
     return assert.equal(Audit.audit(inputs).value, false);
   });
 
+  it('fails when a manifest contains an icon with no size', () => {
+    const manifestSrc = JSON.stringify({
+      icons: [{
+        src: 'icon.png'
+      }]
+    });
+    const manifest = manifestParser(manifestSrc).value;
```

Comment:
You're redefining `manifest`; we should probably move the "good manifest" declared at the top into each test that needs it, or move these out to a set of other bad manifest files that can be included in the same way as the good one.